tiny-example/model_description.yaml

```diff
@@ -0,0 +1,19 @@
+kind: Model
+apiVersion: v1alpha1
+metadata:
+  displayName: Hello Model
+  name: hello-model
```

Comment:
I think this name should match the name of the directory this is contained in and also the name given to the built docker image. I was thinking of updating the other model to be `fibonacci model` throughout.

Reply:
or maybe `fibonacci model` for directory and Readme and then just Fibonacci in the definition

Reply:
The folders were supposed to be a list of what the examples are for, not necessarily what they do (a bit like how we name tests) "Simple-example", "simple-example-using-R", "complex-model-with-file-input". That kind of thing. But inside the folder, they are complete models that can exist outside of these examples.

Reply:
This is a very good point and would be a nice thing to have, especially if they are viewed in isolation in this repo. I ran into the issue of there being a bit of an odd disconect when trying to reference them from the user facing documentation, but maybe I should think a bit harder about how that could be handled in a better way to leave these directories with the `simple-example` type names...

Reply:
Would there be any merit in combining the two and having something like `simple-example--fibonacci-model`?

Reply:
Actually sounds good!